src/earn/poolInfoScreen/Cards.tsx

```diff
@@ -304,23 +311,29 @@ export function AgeCard({
   return (
     <View style={styles.card} testID="AgeCard">
       <View style={styles.cardLineContainer}>
-        <View style={styles.cardLineLabel}>
+        <View style={styles.flexLabel}>
           <LabelWithInfo
             onPress={onInfoIconPress}
             label={t('earnFlow.poolInfoScreen.ageOfPool')}
-            labelStyle={styles.cardTitleText}
+            labelStyle={styles.lineLabel}
             testID="AgeInfoIcon"
+            numberOfLines={undefined}
           />
         </View>
-        <Text style={styles.cardTitleText}>{formattedDuration(dateInterval)}</Text>
+        <View style={styles.flexValue}>
```

Comment:
Do we need this nesting? Here and in a few other places. Just adding `flex: 1` to `styles.lineValue` here appears to yield the same result.

Reply:
This was wrapped with a view for clarity of layout behavior, but you're right. We could add `flex: 1` to the following styles to achieve the same result: `lineValue,` `cardValueText,` and `depositAndEarningsCardValueText.`

Reply:
Changed in fd54594b95927ee10f65a25eef60b1b92c9ced33!